upload-cloud-assembly/action.yml

```diff
@@ -26,16 +26,12 @@ runs:
         INPUT_AWS_S3_BUCKET_NAME: ${{ inputs.aws-s3-bucket-name }}
         INPUT_CDK_APP_DIR: ${{ inputs.cdk-app-dir }}
       run: |
-        cdk_assembly_dir="${INPUT_CDK_ASSEMBLY_DIR:-$GITHUB_WORKSPACE}"
-        cd "$cdk_assembly_dir"
-        tmp_dir="/tmp/${GITHUB_ACTION:-$(date -u +%s)}"
-        mkdir -p "$tmp_dir"
-        cloud_assembly_archive="$tmp_dir/cloud-assembly.zip"
-        if [ ! -f "cdk.json" ]; then
-          echo "No cdk.json file found in '$input_cdk_assembly_path'. You may need to set the 'cdk-app-dir' input."
+        cloud_assembly_dir="${INPUT_CDK_APP_DIR:-$GITHUB_WORKSPACE}"
+        if [ ! -f "${cloud_assembly_dir}/cdk.json" ]; then
+          echo "No cdk.json file found in '$cloud_assembly_dir'. You may need to set the 'cdk-app-dir' input."
           exit 1
         fi
 
         bash "$GITHUB_ACTION_PATH/action.sh" \
           --aws-s3-bucket-name "$INPUT_AWS_S3_BUCKET_NAME" \
-          --cloud-assembly-archive "$cloud_assembly_archive"
+          --cloud-assembly-dir "$cloud_assembly_dir"
```

Comment:
I think it's more correct if we actually name this `cdk-app-dir`. We want a directory to a CDK application, so that we can synthesize the CDK application into a Cloud Assembly (i.e., the contents of `cdk.out`).

Reply:
https://github.com/capralifecycle/actions-lib/pull/6/commits/8e408333e7dacab4f171ff5062257ea3136c487f